Board.java

```diff
@@ -0,0 +1,258 @@
+/*
+ * These files contain the 2048 game logic and method.
+ * Created By : Sachin Gupta
+ * Created Date : 21/9/2024
+ */
+
+package Assignment_5;
+
+import Assignment_5.Constant;
+
+import java.util.Random;
+
+public class Board {
+    Constant constant = new Constant();
+    static final int SIZE = 4;
+    private int[][] board;
+    private Random random;
+    public static int score;
+    public static int moves;
+    boolean hasWon;
+
+    public static final String RESET = "\u001B[0m";
+    public static final String BOLD = "\u001B[1m";
+
+    //Board method initialized the Board with initial values.
+    public Board() {
+        board = new int[SIZE][SIZE];
+        random = new Random();
+        score = 0;
+        moves = 0;
+        hasWon = false;
+        addRandomTile();
+        addRandomTile();
+    }
+
+    //printBoard method print the board of 2048 game.
+    int[][] printBoard() {
+        System.out.println(BOLD + constant.GameTitle + RESET);
+        System.out.println(constant.Score + score + "   " + constant.Moves + moves);
+        System.out.println();
+        System.out.println("+-----+-----+-----+-----+");

```

Comment:
Hardcoded value must be in constant file

Reply:
But I think this is not considered as a hard-coded values